packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx

```diff
@@ -306,10 +344,14 @@ export class CommandBar extends BaseComponent<ICommandBarProps, ICommandBarState
       let item = this.props.items[i];
 
       if (!this._commandItemWidths[item.key]) {
-        let el = this.refs[item.key] as HTMLElement;
-
-        if (el) {
-          this._commandItemWidths[item.key] = el.getBoundingClientRect().width;
+        if (item.itemType === ContextualMenuItemType.Divider) {
+          this._commandItemWidths[item.key] = 1;
```

Comment:
We should still measure it here; there may be padding/margin; and if we ever change the width, then this calculation will be wrong.